RIGImageGallery/RIGSingleImageViewController.swift

```diff
@@ -8,34 +8,44 @@
 
 import UIKit
 
-class RIGSingleImageViewController: UIViewController {
+open class RIGSingleImageViewController: UIViewController {
 
-    var viewerItem: RIGImageGalleryItem? {
+    open var viewerItem: RIGImageGalleryItem? {
         didSet {
             viewerItemUpdated()
         }
     }
 
-    let scrollView = RIGAutoCenteringScrollView()
+    open let scrollView = RIGAutoCenteringScrollView()
+    open let activityIndicator: UIActivityIndicatorView = {
```

Comment:
#? In the case that someone wanted a different style loading indicator, can this be made a `var` so it can be replaced in `loadView`?